src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -438,7 +438,15 @@ public function testAssetHelperWhenAssetsAreEnabled()
 
     public function testAssetHelperWhenTemplatesAreEnabledAndAssetsAreDisabled()
     {
-        $container = $this->createContainerFromFile('assets_disabled');
```

Comment:
shouldn't we remove also `src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/assets_disabled.*`?